lib/ui/src/components/sidebar/Menu.stories.tsx

```diff
@@ -37,28 +37,50 @@ const DoubleThemeRenderingHack = styled.div({
   },
 });
 
+const ExpandedMenu: FunctionComponent<{ menu: any }> = ({ menu }) => (
```

Comment:
Not quite sure of the best way to do this `menu` type. Seems like `menu` is a memoized array of memoized functions, which sounds a bit complicated. My typescript is not quite up to par on that one. Open to suggestions.